lib/src/status.rs

```diff
@@ -305,18 +308,106 @@ pub(crate) async fn status(opts: super::cli::StatusOpts) -> Result<()> {
     let legacy_opt = if opts.json {
         OutputFormat::Json
     } else {
-        OutputFormat::Yaml
+        if  std::io::stdout().is_terminal() {
+            OutputFormat::HumanReadable
+        } else {
+            OutputFormat::Yaml
+        }
     };
     let format = opts.format.unwrap_or(legacy_opt);
     match format {
         OutputFormat::Json => serde_json::to_writer(&mut out, &host).map_err(anyhow::Error::new),
         OutputFormat::Yaml => serde_yaml::to_writer(&mut out, &host).map_err(anyhow::Error::new),
+        OutputFormat::HumanReadable => human_readable_output(&mut out, &host),  
     }
     .context("Writing to stdout")?;
 
     Ok(())
 }
 
+#[derive(Default, Iterable)]
+struct ImageOpts {
+    version: String,
+    transport: String,
+    digest: String,
+}
+
+fn human_readable_output(mut out: impl Write, host: &Host) -> Result<()> {
+    for (status_string, status) in [
+        ("staged", &host.status.staged),
+        ("booted", &host.status.booted),
+        ("rollback", &host.status.rollback),
+    ] {
+        if let Some(host_status) = status {
+            let mut image_opts: ImageOpts = ImageOpts::default();
+            if let Some(image) = &host_status.image {
+                out.write_all(format!("Current {:?} image: {:?}\n", status_string, image.image.image).as_bytes())?;
```

Comment:
Instead of `out.write_all(format!(...))` use `write!` which is both shorter and saves allocating a temporary string.

Also, we can avoid the literal `\n` with `writeln!`.